nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/TestAutomaticIndexing.java

```diff
@@ -101,6 +101,7 @@
 @Deploy("org.nuxeo.ecm.platform.tag")
 @Deploy("org.nuxeo.ecm.automation.core")
 @Deploy("org.nuxeo.elasticsearch.core.test:elasticsearch-test-contrib.xml")
+// @WithFrameworkProperty(name = RECURSIVE_INDEXING_USING_BULK_SERVICE_PROPERTY, value = "true")
```

Comment:
Why keeping it as comment?

Reply:
just to keep in mind that this suite can also be run with a different implementation